lib/OrdinaryDiffEqFIRK/src/firk_tableaus.jl

```diff
@@ -415,13 +414,13 @@ function BigRadauIIA13Tableau(T1, T2)
     c[7] = big"1.0"
 
     e = Vector{T1}(undef, 7)
-    e[1] = big"-0.171003707892600662399316289094713451418682228802554671094543075316220821099884263713032871578607576486535539488632407200766379971279557791263375813632287147"
-    e[2] = big"0.0934967172358652400317534533028674569308657324394316331629203486361371292312231403973668315582870547753526899857449840409175610009437530537219068836721686211"
-    e[3] = big"-0.0538908303114758775848180855518003793385120454908028879947132475960997563222416509683480350817114056356343433378213334826358824351525243402758389616051172681"
-    e[4] = big"0.03036786965048439581219923157368590250090409822952169396779792168990510618756404452728392288892998298088147691907128776816545685599760715439221674418662785"
-    e[5] = big"-0.0169792974425458224044481617230998766694942329759644144506911809530904808476835739189122151558601434810772520378036293579816345384682687602578758514350075723"
-    e[6] = big"0.00942688256820236884916415666439281573527695349338346787075550606528435808025071461023926432308932314282041885090975780812273515256740094297311541275151861292"
-    e[7] = big"-0.00331409873565629283448601269346047459594635696619041493081994712789731442072563377354737487903843138987115421498455722938021358621090485566426506726181005806"
+    e[1] = big"-54.37443689412861451458"
+    e[2] = big"7.000024004259186512041"
+    e[3] = big"-2.355661091987557192256"
+    e[4] = big"1.132289066106134386384"
+    e[5] = big"-0.6468913267673587118673"
+    e[6] = big"0.3875333853753523774248"
+    e[7] = big"-0.1428571428571428571429"
```

Comment:
What happened to the other digits?

Reply:
Those were Hairer's numbers, because we were having trouble figuring out how to derive those coefficients so we used his as a placeholder for testing purposes and they only go up to that specificity. I'm working on figuring out the derivation manually right now, so hopefully we get that done soon.